### PR TITLE
Correct path of route files

### LINKED ISSUE #159: Create routing rules instructions is incorrect in 2.1
image

must be 

az spring gateway route-config create     --name ${FRONTEND_APP}     --app-name ${FRONTEND_APP}     --routes-file ./azure-spring-apps-enterprise/resources/json/routes/frontend.json

### LINKED ISSUE #160: Create routing rules instructions are incorrect in 5
image

routes-file paths are incorrect

must be

az spring gateway route-config create \
    --name ${CART_SERVICE_APP} \
    --app-name ${CART_SERVICE_APP} \
    --routes-file ./azure-spring-apps-enterprise/resources/json/routes/cart-service.json
    
az spring gateway route-config create \
    --name ${ORDER_SERVICE_APP} \
    --app-name ${ORDER_SERVICE_APP} \
    --routes-file ./azure-spring-apps-enterprise/resources/json/routes/order-service.json

az spring gateway route-config create \
    --name ${CATALOG_SERVICE_APP} \
    --app-name ${CATALOG_SERVICE_APP} \
    --routes-file ./azure-spring-apps-enterprise/resources/json/routes/catalog-service.json

### PR DESCRIPTION
**Pull request by dingmeng-xue**
Correct path of rout files.
Fix typos in docs.

### CODE REVIEW

File: azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/full/09-hol-3.1-deploy-backend-apps/README.md [hunk 1]
```diff
@@ -60,17 +60,17 @@ Routing rules bind endpoints in the request to the backend applications. For exa
...
     --name ${ORDER_SERVICE_APP} \
     --app-name ${ORDER_SERVICE_APP} \
-    --routes-file ./routes/order-service.json
+    --routes-file ./azure-spring-apps-enterprise/resources/json/routes/order-service.json
 
 az spring gateway route-config create \
     --name ${CATALOG_SERVICE_APP} \
     --app-name ${CATALOG_SERVICE_APP} \
-    --routes-file ./routes/catalog-service.json
+    --routes-file .=./azure-spring-apps-enterprise/resources/json/routes/catalog-service.json
```
Comment (jiec-msft) [inline azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/full/09-hol-3.1-deploy-backend-apps/README.md]: Shall we remove `=.`?

Comment (dingmeng-xue) [reply azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/full/09-hol-3.1-deploy-backend-apps/README.md]: Thanks. Fixed it.

**Comment by anishp55** [comment]
i've updated the README for the first intermediate lab so that they are already in the `acme-fitness-store/azure-spring-apps-enterprise` directory

**Review by anishp55 — CHANGES_REQUESTED** [review]
please remove the `./azure-spring-apps-enterprise` and make it `./resources`
one of the PRs puts us into the `azure-spring-apps-enterprise` directory

**Pull request merged**